rca/project_styleguide/templates/patterns/molecules/navigation/primarynav.html

```diff
@@ -1,35 +1,122 @@
 {% load wagtailcore_tags %}
-<nav role="navigation" aria-label="Primary">
-    <ul class="nav nav--primary" role="menubar">
-        {% for link in primarynav %}
-            <li class="" role="menuitem">
-                <a href="{{ link.url }}" class="">
-                    <span class="">{{ link.text }}</span>
-                </a>
+<div class="header__inner header__inner--level-zero">
+    {% include 'patterns/atoms/logo/logo.html' %}
+    {% include 'patterns/molecules/menu-toggle/menu-toggle.html' with modifier="header__menu-toggle--small" %}
+    {% include 'patterns/atoms/back-link/back-link.html' %}
+    {% include 'patterns/molecules/navigation/standoutnav.html' with modifier="nav--standout-desktop" %}
+    <div class="app__search app__search--tablet">
+        {% include 'patterns/molecules/search/search.html' %}
+    </div>
+</div>
+<span class="header__inner header__inner--container">
+    <div class="header__navigation-toggle">
+        {% include 'patterns/molecules/menu-toggle/menu-toggle.html' with modifier="header__menu-toggle--medium" %}
+        {% include 'patterns/molecules/search-toggle/search-toggle.html' %}
+
+        <a class="header__menu-close" href="#" data-close-menu>
+            <svg width="14" height="14"><use xlink:href="#close"></use></svg>
+        </a>
+    </div>
+
+    <span class="header__inner--menu-container">
```

Comment:
should we have `header__inner` here as well?

Reply:
aye 👍